### PR TITLE
Pv2 370 365 361 519 608 340 new and previous d lock passing test

### PR DESCRIPTION
**Pull request by samuel-dev-dba**

### CODE REVIEW

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/Levy learner goes on a planned break - PV2-296.feature [hunk 1]
```diff
@@ -101,8 +101,6 @@ And only the following provider payments will be generated
 
 Examples: 
         | Collection_Period         |
-        | R04/Current Academic Year |
```
Comment (carlbm) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/Levy learner goes on a planned break - PV2-296.feature]: Should these have been removed?

Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/Levy learner goes on a planned break - PV2-296.feature]: Now Restored

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Employer Change of Circumstances/One Levy learner,Employer change and change to the Negotiated Cost earlier than expected_PV2-361.feature [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	| pe-1          | 12000                | 01/Aug/Current Academic Year        | 3000                   | 01/Aug/Current Academic Year          | 0                       |                                        | 0                         |                                          | 90%                         |
+	| pe-2          | 12000                | 01/Aug/Current Academic Year        | 3000                   | 01/Aug/Current Academic Year          | 5000                    | 25/Oct/Current Academic Year           | 625                       | 25/Oct/Current Academic Year             | 90%                         |
+
+When the amended ILR file is re-submitted for the learners in collection period <Collection_Period>
+	
+Then the following learner earnings should be generated
+	| Delivery Period           | On-Programme | Completion | Balancing | Price Episode Identifier |
+	| Aug/Current Academic Year | 1000         | 0          | 0         | pe-1                     |
+	| Sep/Current Academic Year | 1000         | 0          | 0         | pe-1                     |
+	| Oct/Current Academic Year | 1000         | 0          | 0         | pe-2                     |
```
Comment (ESFA-Pawan) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Employer Change of Circumstances/One Levy learner,Employer change and change to the Negotiated Cost earlier than expected_PV2-361.feature]: On-Programme amount should be 450 for Oct/current academic year

**Comment by ESFA-Pawan** [comment]
@samuel-dev-dba as discussed please update PV2-296 as per new spec in:
https://github.com/SkillsFundingAgency/das-payments-V2-spec/tree/InProgressTests/End-to-End%20-%20Business%20Acceptance%20Tests/Levy%20-%20Learner%20Change%20of%20Circumstances

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Employer Change of Circumstances/One Levy learner changes employer,incentive earned in transfer month, ILR transfer is before the commitment changes_PV2-374.feature [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+And price details as follows
+    | Price Episode Id | Total Training Price | Total Training Price Effective Date | Total Assessment Price | Total Assessment Price Effective Date | Residual Training Price | Residual Training Price Effective Date | Residual Assessment Price | Residual Assessment Price Effective Date | SFA Contribution Percentage |
+    | pe-1             | 6000                 | 06/Aug/Current Academic Year        | 1500                   | 06/Aug/Current Academic Year          | 0                       |                                        | 0                         |                                          | 90%                         |
+    | pe-2             | 6000                 | 06/Aug/Current Academic Year        | 1500                   | 06/Aug/Current Academic Year          | 4000                    | 09/Nov/Current Academic Year           | 1625                      | 09/Nov/Current Academic Year             | 90%                         |
+When the amended ILR file is re-submitted for the learners in collection period R04/Current Academic Year
+Then the following learner earnings should be generated
```
[the remark this answers is not in the document]
Comment (ESFA-Pawan) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Employer Change of Circumstances/One Levy learner changes employer,incentive earned in transfer month, ILR transfer is before the commitment changes_PV2-374.feature]: New version is now correct

**Comment by ESFA-Pawan** [comment]
@samuel-dev-dba 296 and 374 are now fine.

**Pull request merged**